language-plutus-core/test/types/fold.plc.golden

```diff
@@ -0,0 +1 @@
+Error at 10:19. Variable a is not in scope.
```

Comment:
Is this supposed to happen?

Reply:
Nope. Shoot. I'll change this to WIP.

Reply:
Oh, perhaps this test case is in error. The `a` that is bound is a type variable. I will ping Chad.